src/cprover/cprover_parse_options.cpp

```diff
@@ -277,6 +277,7 @@ int cprover_parse_optionst::main()
 
     solver_options.trace = cmdline.isset("trace");
     solver_options.verbose = cmdline.isset("verbose");
+    solver_options.large_step = cmdline.isset("large-step");
```

Comment:
Please add a `help` entry (in `cprover_parse_optionst::help` at the bottom of this file).